scipy/stats/_stats_py.py

```diff
@@ -534,7 +534,7 @@ def mode(a, axis=0, nan_policy='propagate', keepdims=False):
     return ModeResult(modes[()], counts[()])
 
 
-def _put_nan_to_limits(a, limits, inclusive):
+def _put_val_to_limits(a, limits, inclusive, val=np.nan, xp=None):
     """Put NaNs in an array for values outside of given limits.
```

Comment:
```suggestion
    """Put `val` in an array for values outside of given limits.
```

Reply:
also could you update the rest of the docstring please

Reply:
OK.... done.

Reply:
Thanks! Could you either split this up or update the description with the additional changes, both are fine with me

Reply:
Updated description. Thanks!